src/header.tsx

```diff
@@ -173,47 +174,69 @@ export const HeaderLayout: React.FC<HeaderLayoutProps> = ({
-                    </Stack>
-                </Box>
-            ) : sideBySideExtraOnDesktop ? (
-                <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
-                    <GridLayout
-                        template="6+6"
-                        left={
+        <div>
+            <ResponsiveLayout
+                isInverse={isInverse}
+                dataAttributes={{'component-name': 'HeaderLayout', ...dataAttributes}}
+            >
+                <div style={{paddingBottom: bleed && isInverse && extra ? 40 : 0}}>
+                <OverscrollColor />
+                {isTabletOrSmaller ? (
+                    <div>
+                        <Box paddingTop={header ? 32 : 0} paddingBottom={24}>
+                            <Stack space={24}>
+                                {header}
+                            </Stack>
+                        </Box>
+                    </div>
+                ) : sideBySideExtraOnDesktop ? (
+                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={bleed && isInverse ? 8 : 48}>
+                        <GridLayout
+                            template="6+6"
+                            left={
+                                <Stack space={32}>
+                                    {breadcrumbs}
+                                    {header}
+                                </Stack>
+                            }
+                            right={extra}
+                        />
+                    </Box>
+                ) : (
+                    <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={32}>
+                        <Stack space={isTabletOrSmaller ? 24 : 32}>
                             <Stack space={32}>
                                 {breadcrumbs}
                                 {header}
                             </Stack>
-                        }
-                        right={extra}
-                    />
-                </Box>
-            ) : (
-                <Box paddingTop={breadcrumbs ? 16 : 48} paddingBottom={48}>
-                    <Stack space={isTabletOrSmaller ? 24 : 32}>
-                        <Stack space={32}>
-                            {breadcrumbs}
-                            {header}
                         </Stack>
-                        {extra}
-                    </Stack>
-                </Box>
+                    </Box>  
+                )}
+                </div>
+            </ResponsiveLayout>
+            {extra && (isTabletOrSmaller || !sideBySideExtraOnDesktop) && (
+                <ResponsiveLayout 
+                    isInverse={!!(isInverse && !bleed)}
+                    dataAttributes={{'component-name': 'HeaderLayout', ...dataAttributes}}
+                >
+                    <div style={{
+                        transform: bleed && isInverse ? `translateY(-40px)` : `translate(0, 0)`,
+                        marginBottom: bleed && isInverse ? '-40px' : '0px',
```

Comment:
I'd prefer a solution without transform translate nor negative margins, like the one I proposed

Reply:
I understand. If possible, look at commit edits on this branch (https://github.com/Telefonica/mistica-web/tree/vivo-header-bleed-alt). In case it seems more interesting, I can open the PR of this one. It is similar to the process you indicated. I believe it is much cleaner and without changing as many code properties as desired. The other two changes mentioned now, I can also apply there.

Reply:
In the case of the aforementioned branch (vivo-header-bleed-alt), I can make some changes so that if the bleed is false, I can automatically get the extra in the first ResponsiveLayout. If the prop bleed is activated, it will pull the code snippet with the second ResponsiveLayout.